tests/ProxyManagerTest/ProxyGenerator/Util/ProxiedMethodsFilterTest.php

```diff
@@ -59,8 +55,8 @@ static function (ReflectionMethod $method) : string {
     }
 
     /**
-     * @param string[]|null $excludes
-     * @param string[]      $expectedMethods
+     * @param array<int, string>|null $excludes
+     * @param string[]                $expectedMethods
```

Comment:
should we use `<>` instead of `[]` to express arrays types?

Reply:
When the type of the key is important, yes